Added a simple health ui and color coding for the robots

DerApplaudeur opened the pull request:
We now have a health ui and the Robots have different colors.
For now the only way to take damage is pressing f which only damages the player controlled robot, mainly to test it the value would update correctly.

Any ideas for visual changes for this ui are more than welcome, because right now it's ugly :)

DerApplaudeur commented:
The position of the health display is static for each robot since only the final robot knows the total amount of robots which makes a dynamic system within the robot class impossible (i think).
If we were to move this to  game.py this wouldn't be a problem but i don't think thats necessary.

Luceleven reviewed (approved):
Works fine, but is indeed ugly. Would help a bit if the health ui is at the top of the screen, but for now it's okay.

DerApplaudeur commented:
I moved the ui to the top for now, if we add an options menu we can add the option to choose the position either freely or from a number of pre-determined options.
I also changed the color of the rectangle surrounding the health counter to a , in my opinion, nicer looking color than the previous black.
